style.css

```diff
@@ -112,3 +112,52 @@ h1 {
   margin-top: 0px;
   margin-bottom: 24px;
 }
+
+/* popular destinations section */
+.popular-destinations {
+  background-image: url(./images/destinations_background.png);
+  background-repeat: no-repeat;
+  height: 1053px;
+}
+
+h2 {
+  text-align: center;
+  color: #ffffff;
+  font-weight: bold;
```

Comment:
`h2``font-weight`by default jau bus `bold`, nebereikia to nurodyti.
https://www.w3schools.com/cssref/css_default_values.php